### DISCUSSION TITLE
mediamtx_v1.1.0_windows_amd64 Questions

### DISCUSSION THREAD

**Discussion by CodeMiragecode** [bluenviron/mediamtx#2380] [Questions]
Hello!

Please tell me how to register the service for windows Version

**Comment by liusuyi2021** [comment]
请认真看文档 文档都有说明如何配置系统服务

**Comment by aler9** [comment]
instructions on how to register MediaMTX as a service on Windows are in the README:

https://github.com/bluenviron/mediamtx#start-on-boot

[a comment by an automated account is not shown]
